### ISSUE TITLE
Symfony 6 compatibility

### ISSUE THREAD

**Issue by studio-m2v** [dmishh/SettingsBundle#124]
Hi, 
Has anyone already started working on the changes to be made to be compatible with Symfony 6?
Thank you

**Comment by rvanlaak** [comment]
Yes this bundle deserves some love. Would you be willing to help with doing maintenance PRs?

**Comment by studio-m2v** [comment]
@rvanlaak I will try to work on it

**Comment by tacman** [comment]
What do you think about dropping support for all EOL components for version 3?  That'd make it much easier to upgrade.

That is, PHP 7.4+, Symfony 5.4|6.0, etc.

Personally, I'd like to only support PHP 8, PHP 7.4 is in security fixes only, and in 3 months it will be officially EOL.  Given that anyone who is using older version can still use v2, what do you think?  

Related: creating a bundle configuration is MUCH easier in Symfony 6.1.  

I can jump in and bump this to PHP 8 if you're okay with making that leap.
